test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -99,6 +101,15 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, appHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  preloadedTable.policy_table.app_policies[appId].AppHMIType = { appHMIType }
```

Comment:
```suggestion
  preloadedTable.policy_table.app_policies[appId].AppHMIType = { appHMIType }
  preloadedTable.policy_table.app_policies[appId].enabled = true
```
Add mandatory parameter for WebEngine app

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-7b7ca1c3b1b899ba16ded7d51adceea0R109)